### DISCUSSION TITLE
Linux logs via Filebeats

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#8946] [Unsupported Versions] [answered]
Hello everyone! 

Currently, I am trying to send Linux logs to Security Onion (v2.3.180) via filebeats. It should be as simple as installing filebeats, modifying the yml file, and opening logstash to the endpoint. Is anyone else ingesting logs from Linux endpoints via filebeats?

**Comment by dougburks** [comment]
Is it not working for you? Have you checked the logs for additional clues as to why it's not working?

**Reply by ColeVan** [reply]
Hello Doug,

Unfortunately it is not working. 

Running a filebeat -c filebeat.yml test output results in a successful test connection to the Security Onion manager. However, once I open the firewall and allow it in and restart filebeats I see no connection at all. Have you tested filebeats on linux yet? If so, could you let me know if it works?

**Reply by dougburks** [reply]
Seems to work fine for me. 

Did you download Filebeat from your Security Onion manager?
https://docs.securityonion.net/en/2.3/beats.html

Are you running Security Onion in Standalone or Distributed mode and have you verified that Logstash is running?
https://docs.securityonion.net/en/2.3/logstash.html

Are you running `so-allow` and choosing the `b` option?
https://docs.securityonion.net/en/2.3/so-allow.html

Have you enabled one or more inputs in `filebeat.yml` and configured the logstash output?

Have you checked the logs for additional clues as to why it's not working?

**Comment by ColeVan** [comment]
Hello Doug,

I made a few changes in yaml file and its working now. The input was set to false so I changed it to true. Is there a way to properly parse filebeat logs from linux hosts? The logs do not seem right when streaming in.

![image](https://user-images.githubusercontent.com/70167373/197285298-dc28eefb-a102-45e2-9812-21dc4b32d0e2.png)

**Comment by ColeVan** [comment]
@dougburks 

My apologies for not answering your original question. Here is what I'm using.

Standalone SO2.3.180

Logstash has been restarted 

so-allow was ran to allow endpoint in

I used the deb package from SOC downloads tab.

**Comment by ColeVan** [comment, marked as answer]
I finally got everything working. I used my google foo and found a way to parse the linux  sysmon data. Everything is coming in great!
![image](https://user-images.githubusercontent.com/70167373/197317016-76202088-a517-48fb-85f2-d9e1054c0507.png)
